components/nimbus/metrics.yaml

```diff
@@ -27,6 +27,9 @@ nimbus_events:
       enrollment_id:
         type: string
         description: A unique identifier generated at enrollment time
+      experiment_type:
+        type: string
+        description: Indicates whether this is an experiment or rollout
```

Comment:
Should experiment_type be used anywhere at this time or are we still waiting for rollouts to be released?

Reply:
It won't be used until rollouts are released. Even if we don't use the type yet, it will be ready when rollouts are.